unstract/core/src/unstract/core/metrics/backends/queue.py

```
@@ -0,0 +1,215 @@
+                    content_type="application/json",
+                    content_encoding="utf-8",
+                    headers={
+                        "task": self.task_name,
+                        "id": str(uuid.uuid4()),
+                    },
+                )
+
+            logger.debug(
+                f"Queued metric task: {event.metric_name} for org {event.org_id}"
+            )
+            return True
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)